### DISCUSSION TITLE
Directory structure

### DISCUSSION THREAD

**Discussion by akihikokuroda** [i-am-bee/bee-community-tools#13] [General]
As the community tool repository, I think that files for each tool are better under its own directory instead of spreading them in examples, src, and tests directories in the repository.
How about directory structure something like this?  
```
/tools
  /version.ts
  /version.test.ts
  /xxxx
    /README.md
    /version.ts
    /version.test.ts
    /src
      /xxxx.ts
      /xxxx.test.ts
    /examples
      /xxxx.ts
    /tests
      /xxxx.test.ts
```
It's probably better each tool has the framework version and its own versitn separately.

**Comment by grahamwhiteuk** [comment]
Yes, as the collection grows the current approach may become quite unwieldy.  With a small number of tools, as things stand at the moment, it's manageable.  Would you like to PR the change in structure?

**Reply by akihikokuroda** [reply]
Yes, I will.  Thanks!
